src/data/compositors/mutter.json

```diff
@@ -70,6 +98,10 @@
       "interface": "xdg_wm_dialog_v1",
       "version": 1
     },
+    {
+      "interface": "xx_session_manager_v1",
+      "version": 1
+    },
```

Comment:
I was not sure about this one, which is experimental and disabled by default, and have it explicitly enabled on my env cause I'm working on supportting it Chromium. Should I keep it?

Reply:
Yes, I think it makes sense to keep it even if it's not enabled by default.